components/error-summary.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: component-detail
+group: components
+
+title: Error summary
+description: Error summary is used to present validation errors of a form.
+status: Beta
+
+github_link: https://github.com/apluslms/a-plus-design-system/issues/9
```

Comment:
This URL should point to an issue, as the PR will be closed. You could create an empty issue for further discussions about the component, as I have done for the previous components.